babel.config.js

```diff
@@ -69,7 +71,7 @@ module.exports = function (api) {
 
   let transformRuntimeOptions;
 
-  const nodeVersion = bool(process.env.BABEL_8_BREAKING) ? "14.17" : "6.9";
+  const nodeVersion = BABEL8 ? "14.17" : "6.9";
```

Comment:
Which rule is this?

Reply:
There are no rules about this, it's just something I do in passing when modifying this file.

Reply:
Mh ok, I would prefer to keep the `bool(process.env.BABEL_8_BREAKING)` inline.

The plugin that strips away the `BABEL_8_BREAKING` flag at build time can replace `bool(process.env.BABEL_8_BREAKING) ? "14.17" : "6.9"` with `"6.9"`, but it cannot replace `BABEL8 ? "14.17" : "6.9"`. It doesn't really matter in this file since it's not transformed, but since it matters almost everywhere else I would prefer to keep consistency.

Reply:
Also, if we are consistent everywhere it will be very easy to codemod the flag away using the plugin we already have :)

Reply:
Done!